setup.py

```diff
@@ -15,11 +18,22 @@
     "moto==1.3.13",
 ]
 
+# Import README for long-description.
+with open(here / "README.md") as f:
+    long_description = f.read()
+
+# Load the package's __version__.py module as a dictionary.
+vars_from_version_file = {}
+with open(here / package_name / "__version__.py") as f:
+    exec(f.read(), vars_from_version_file)
```

Comment:
Do we really need an exec there? Can't we just import it?

Reply:
Yep, I will fix it